test/e2e/apps/disruption.go

```diff
@@ -151,7 +147,7 @@ var _ = SIGDescribe("DisruptionController", func() {
 			}
 			createPodsOrDie(cs, ns, c.podCount)
 			if c.replicaSetSize > 0 {
-				createReplicaSetOrDie(cs, ns, c.replicaSetSize, c.exclusive)
```

Comment:
isn't this changing the meaning of the test?  by using exclusive here you are no longer testing that exclusivity of host ports is preserved for disruption

Reply:
I thought there is no point in preserving exclusivity for container image which doesn't need port at all. Previously we used to do that because we were using echoserver container image.

I will wait for others from @kubernetes/sig-apps-misc to weigh in here.
/cc @mfojtik